settings.yaml

```diff
@@ -0,0 +1,40 @@
+ConfirmationCodesService:
+    Db:
+        LogsConnString:
+            settings-key: ConfirmationCodesService-LogsConnString
+            types:
+                - AzureTableStorage
+        ClientPersonalInfoConnString:
+            settings-key: PersonalInfoConnString
+            types:
+                - AzureTableStorage
+    DeploymentSettings:
+        IsProduction:
+            settings-key: ConfirmationCodesService-IsProduction
```

Comment:
should this be a setting?

Reply:
Yes. It's used for switching between real sms/email and their mocks.